views/doc.cfm

```diff
@@ -4,21 +4,25 @@
 			<h1 id="docname">#data.name#</h1>
 			<p>#autoLink(data.description)#</p>
 			<cfif StructKeyExists(data, "syntax") AND Len(data.syntax)>
-				<p id="syntax"><cfif data.type IS "tag"><small><span class="glyphicon glyphicon-tags" title="Tag Syntax"></span></small> &nbsp;</cfif><code>#Replace(encodeForHTML(data.syntax), Chr(10), "<br>", "ALL")#<cfif data.type IS "function" AND StructKeyExists(data, "returns") AND Len(data.returns)> <em>&##8594; returns #encodeForHTML(data.returns)#</em></cfif></code></p>
+				<p id="syntax">
+					<cfif data.type IS "tag">
+						<small><span class="glyphicon glyphicon-tags" title="Tag Syntax"></span></small> &nbsp;
+					</cfif>
+					<code>#Replace(encodeForHTML(data.syntax), Chr(10), "<br>", "ALL")#
+						<cfif data.type IS "function" AND StructKeyExists(data, "returns") AND Len(data.returns)>
+							<em>&##8594; returns #encodeForHTML(data.returns)#</em>
+						</cfif>
+					</code>
+				</p>
 				<cfif data.type IS "tag">
 					<cfif StructKeyExists(data, "script")>
 						<cfset data.scriptTitle = "Script Syntax">
 					<cfelseif ListFindNoCase("cfabort,cfbreak,cfcontinue,cfreturn,cfexit", data.name)>
 						<cfset data.scriptTitle = "Script Syntax">
-						<cfset data.script = ReplaceNoCase(data.name, "cf", "") & ";">
+						<cfset data.script = replaceScript(name = data.name, mode = "cf") & ";">
```

Comment:
This is incorrect here... For tags like `cfabort` the script eqivilent is simply `abort;` so it should remove the "cf" from the tag name. Given that I don't see any need to add the replaceScript function since it is only used once.

Reply:
I wanted to remove the bunch of cfset tags so you could focus on the major code.

Reply:
Ah I see you are passing a different mode.